spec/sec-string-objects-patch.html

```diff
@@ -1,39 +1,74 @@
 <emu-clause id="sec-properties-of-the-string-prototype-object">
   <h1>Properties of the String Prototype Object</h1>
   <emu-clause id="sec-string.prototype.match">
-    <h1>String.prototype.match ( _regexp_ <ins>[, _mapfn_ ]</ins> )</h1>
-    <p>When the `match` method is called with argument _regexp_ <ins>and optional argument _mapfn_</ins>, the following steps are taken:</p>
+    <h1>String.prototype.match ( _regexp_ <ins>[, _options_ ]</ins> )</h1>
+    <p>When the `match` method is called with argument _regexp_ <ins>and optional argument _options_</ins>, the following steps are taken:</p>
     <emu-alg>
     1. Let _O_ be ? RequireObjectCoercible(*this* value).
-    1. <ins>If _mapfn_ is not *undefined* and IsCallable(_mapfn_) is *false*, throw a *TypeError* exception.</ins>
+    1. <ins>If _options_ is not present, let _options be *undefined*.</ins>
+    1. <ins>Let _matchOptions_ be ? ToMatchOptions(_options_).</ins>
     1. If _regexp_ is neither *undefined* nor *null*, then
       1. Let _matcher_ be ? GetMethod(_regexp_, @@match).
       1. If _matcher_ is not *undefined*, then
-      1. Return ? Call(_matcher_, _regexp_, &laquo; _O_<ins>, _mapfn_</ins> &raquo;).
+        1. Return ? Call(_matcher_, _regexp_, &laquo; _O_<ins>, ! FromMatchOptions(_matchOptions_)</ins> &raquo;).
```

Comment:
… and then this would be
```suggestion
        1. Return ? Call(_matcher_, _regexp_, &laquo; _O_<ins>, _matchOptions_</ins> &raquo;).
```